CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Change log itself follows [Keep a CHANGELOG](http://keepachangelog.com) format.
 ## Unreleased
 
 ### Added
+* `Faker.Pizza` [@lauraannwilliams][]
```

Comment:
This is just label in markdown, you also need to add URL to your page at the end of changelog.